### ISSUE TITLE
[IMRPOVEMENTS]: 

### ISSUE THREAD

**Issue by Jyad21** [typical-developers/oaklands-feedback#564]
### Roblox Username

Jyad21

### What do you think can have quality-of-life improvements?

Storage Boxes

### How do you think things can be improved?

As of right now the storage boxes have no inventory, I have spent the last few days building an auto storage system, but the storage boxes accept any type of wood aslong as its smooth planked, I think adding something like the filter conveyor filtering will be better, or an option on the boxes to only accept that specific type of item, my auto storage consists of filter conveyors and storage boxes, but it seems my wood storage part just picks up any smooth planks.

**Comment by Zanados** [comment]
I agree.  Please have the drawers accept only one type and state of wood/stone, eg, Oak-Sanded or Marble-Refined.   For me, mostly so that the front of the drawer shows me what is inside, instead of just Stone or Ore.
